### DISCUSSION TITLE
More advanced options for staff

### DISCUSSION THREAD

**Discussion by sjoerddebruin** [OpenRCT2/OpenRCT2#15795] [Feature requests]
- [ ] Change default selected options (mow grass, inspection, default entertainer)
- [ ] Add option for random entertainers
- [ ] Show total costs for each group
- [ ] Add the ability to pay less, with the consequence of working slower
- [ ] Add the option to copy patrol area's

**Comment by IntelOrca** [comment]
> Add the ability to pay less, with the consequence of working slower

I am not fond of adding new game mechanics right now.

**Comment by mrtnptrs** [comment]
This enhancement request should be split up into multiple ones. Also, the first one is partly implemented now (inspection time, mow grass or not). 

I like the second idea personally, but don't know really what you mean with the third one @sjoerddebruin 

Also, the fourth one will really change the game, as this is the behavior projects like CorsixTH have, but I personally don't like that idea.

Fifth option: why would you like to be able to copy patrol areas? What would be the benefit of it? Isn't it illogical to have two persons on the same piece of park?

Maybe @sjoerddebruin could respond to the above questions and then split the enhancement request up into multiple ones? One for each enhancement request.
